packages/snap-controller/src/Recommendation/RecommendationController.test.ts

```diff
@@ -541,6 +541,10 @@ describe('Recommendation Controller', () => {
 		 */
 
 		const result = controller.store.results[0];
+
+		//this tests we send the core data and not the display data
+		result.mask.set({ id: 'custom' });
```

Comment:
This test is fine, as It seems that we were always using `result.id` and not grabbing from `result.display.id`; but we may want to also test a change to the core mappings as well (as that has changed here).